packages/eslint-plugin-dialtone/docs/rules/deprecated-icons.md

```diff
@@ -1,6 +1,6 @@
 # Finds deprecated dialtone svg and vue icons usage (deprecated-icons)
 
-We separated icons from dialtone into dialtone-icons repo, 
+We separated icons from dialtone into dialtone-icons repo,
```

Comment:
```suggestion
We separated icons from dialtone into dialtone-icons package,
```

Reply:
Reworded a bit to make it more clear, thanks!